private-templates-service/client/src/utils/Scroller.tsx

```diff
@@ -0,0 +1,17 @@
+
+export class Scroller {
+  timeStamp: number;
+  constructor() {
+      this.timeStamp = 0;
+  }
+  // TODO: decide which scroll feels more natural
+  verticalScroll = (e: React.WheelEvent) => {
+      e.preventDefault();
+      let item = e.currentTarget;
+      if (e.timeStamp - this.timeStamp < 1000) {
+          return;
+      }
+      this.timeStamp = e.timeStamp;
+      item.scrollTop += 10 * e.deltaY;
+  }
+}
```

Comment:
I think its best to avoid our own scroll logic, though that is a decision for the team to make. We can leave this in for now.

Reply:
I agree, I ran into some issues using the fabric ui scrollable and based this modal after Seva's changes for the tags scolling on the all cards page, I'll see if I can find another way to make it work without the scroll logic in a later PR